### DISCUSSION TITLE
Networking

### DISCUSSION THREAD

**Discussion by johnspeny** [axmolengine/axmol#638] [Q&A] [answered]
I noticed websockets were replaced with yasio. I want to use Nakama client sdk for cocos2d-x in adxe. I do not know how that sdk can be translated to adxe. Any help. https://github.com/heroiclabs/nakama-cocos2d-x

**Comment by rh101** [comment, marked as answer]
Have you tried compiling it with ADXE?  If you need the WebSockets code that exists in cocos2d-x but not ADXE, then just add the cocos2d-x WebSocket.cpp and .h files to your own project, then add the third-party WebSocket library as well, and link them all up in your CMakeLists.txt.

**Reply by johnspeny** [reply]
Why didn't i think of that. Good suggestion 👍.
